pkg/dqlite/controller/client/controller.go

```diff
@@ -7,17 +7,21 @@ import (
 
 	"github.com/canonical/go-dqlite/client"
 	"github.com/canonical/go-dqlite/driver"
+	"github.com/rancher/k3s/pkg/version"
 	controllerv1 "github.com/rancher/wrangler-api/pkg/generated/controllers/core/v1"
 	"github.com/sirupsen/logrus"
 	v1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/labels"
 )
 
+var (
+	nodeID      = "cluster." + version.Program + ".cattle.io/node-id"
+	nodeAddress = "cluster." + version.Program + ".cattle.io/node-address"
```

Comment:
It is almost as if we need a `whitelabel` package that you inject with a minimal number of build-time parameters (having sensible defaults, obv) and an `init()` to wire-up/derive dependent values.